cpp/es1.2/es2_day1.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+
+int main() {
+    vector<int> v;
+    int n; 
```

Comment:
You could move this variable declaration inside the `do`-`while` loop, actually inside the `if (scelta == 1)` block: you don't need it anymore once you pushed it back into the vector.